truenas/webui
# NAS-110948 / 21.08 / Update d3.js to 5.16

The ticket shows an error I was encountering during compile time. Updating d3 and @types/d3 seems make it go away.

To test this:
- Make sure there are no d3 related compile errors
- Make sure all d3 charts continue to function properly.
